dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -303,13 +313,13 @@ Configuration InstallCAM
 				#put java path in front of the oracle defined path
 				if ($NewPath -notlike "*"+$using:JavaBinLocation+"*")
 				{
-				  $NewPath= $using:JavaBinLocation + �;� + $NewPath
+				  $NewPath= $using:JavaBinLocation + ";" + $NewPath
 				}
 
 				#these get added to the environment on next reboot
-				Set-ItemProperty -Path "$Reg" -Name PATH �Value $NewPath
-				Set-ItemProperty -Path "$Reg" -Name JAVA_HOME �Value $using:JavaRootLocation
-				Set-ItemProperty -Path "$Reg" -Name classpath �Value $using:JavaLibLocation
+				Set-ItemProperty -Path "$Reg" -Name PATH -Value $NewPath
```

Comment:
it looks like guithub and linux don't like whatever encoding primesoft is using for the files so the get rendered weird in the brower and on my kubuntu machine. This is just to "correct" it so it looks and runs right. So yes, '-' to '-'